README.md

```diff
@@ -1 +1,131 @@
-# azure-functions-openai-aisearch-python
\ No newline at end of file
+<!--
+---
+name: Azure Function using OpenAI trigger and bindings extension to highlight OpenAI retrieval augmented generation with Azure AI Search.
+description: This repository contains an Azure Function using OpenAI trigger and bindings extension to highlight OpenAI retrieval augmented generation with Azure AI Search. The sample uses managed identity.
+page_type: sample
+products:
+- azure-functions
+- azure
+- entra-id
+urlFragment: azure-functions-openai-aisearch-dotnet8
+languages:
+- csharp
+- bicep
+- azdeveloper
+---
+-->
+
+# Azure Functions
+## Using Azure Functions OpenAI trigger and bindings extension to import data and query with Azure Open AI and Azure AI Search
+
+This sample contains an Azure Function using OpenAI bindings extension to highlight OpenAI retrieval augmented generation with Azure AI Search.
+
+You can learn more about the OpenAI trigger and bindings extension in the [GitHub documentation](https://github.com/Azure/azure-functions-openai-extension) and in the [Official OpenAI extension documentation](https://learn.microsoft.com/en-us/azure/azure-functions/functions-bindings-openai)
+
+
+## Prerequisites
+
+* [.NET 8 SDK](https://dotnet.microsoft.com/download/dotnet/6.0) or greater (Visual Studio 2022 recommended)
```

Comment:
Please steal prereqs from another sample like this one:

https://github.com/Azure-Samples/function-python-ai-openai-chatgpt

Reply:
fixed